packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1969,28 +1969,87 @@ abstract class BuildContext {
   /// [InheritedWidget] subclasses that supports partial updates, like
   /// [InheritedModel]. It specifies what "aspect" of the inherited
   /// widget this context depends on.
+  @Deprecated('Use dependOnInheritedWidgetOfExactType insead')
```

Comment:
Also: The doc comment for this method should just say that this method is deprecated and point to the new method (no need to duplicate the doc comment).

Reply:
Here and elsewhere.

Reply:
Done.

>The doc comment for this method should just say that this method is deprecated and point to the new method (no need to duplicate the doc comment).

I only left the first sentence.